usage/cli/src/main/java/brooklyn/cli/Main.java

```diff
@@ -536,9 +570,10 @@ protected BrooklynLauncher createLauncher() {
             BrooklynLauncher launcher;
             launcher = BrooklynLauncher.newInstance();
             launcher.localBrooklynPropertiesFile(localBrooklynProperties)
-                    .ignorePersistenceErrors(ignorePersistenceErrors)
-                    .ignoreWebErrors(ignoreWebErrors)
-                    .ignoreAppErrors(ignoreAppErrors)
+                    .ignorePersistenceErrors(!startupFailOnPersistenceErrors)
```

Comment:
Should we (in separate PR?) align the method names in launcher with the cli names? e.g. `failOnPersistenceErrors(boolean)`, and deprecate the `ignorePersistenceErrors(boolean)`?